packages/edge/README.md

```diff
@@ -0,0 +1,20 @@
+# `@llamaindex/edge`
+
+Supports llamaindex for Vercel edge runtime.
+
+Limited to the following features:
+
+- Reader (for reading from the file system)
+- Storage (for writing to the file system, and node APIs)
+
+## Usage
```

Comment:
@himself65 @yisding How about adding the usage for edge to the main README and still copying it?